pymc3/distributions/distribution.py

```diff
@@ -376,7 +378,7 @@ def __getattr__(self, name):
             raise AttributeError("'NoDistribution' has no attribute '%s'" % name)
         return getattr(self.parent_dist, name)
 
-    def logp(self, x):
+    def logp(x, *inputs):
```

Comment:
The existence of this method is not in line with actual distributions where this stuff is done through `pm.logp`.
Can it be dispatched?
**EDIT: ** It looks like it _is_ dispatched below?

Also the docstring does not account for all parameters.